Data/HashMap/Base.hs

```diff
@@ -1126,9 +1126,9 @@ adjust# f k0 m0 = go h0 k0 0 m0
         | otherwise = t
 {-# INLINABLE adjust# #-}
 
--- | /O(log n)/  The expression (@'update' f k map@) updates the value @x@ at @k@,
--- (if it is in the map). If (f k x) is @'Nothing', the element is deleted.
--- If it is (@'Just' y), the key k is bound to the new value y.
+-- | /O(log n)/  The expression (@'update' f k map@) updates the value @x@ at @k@
+-- (if it is in the map). If (@f x@) is @'Nothing'@, the element is deleted.
+-- If it is (@'Just' y@), the key @k@ is bound to the new value @y@.
```

Comment:
```suggestion
-- | /O(log n)/  The expression @('update' f k map)@ updates the value @x@ at @k@
-- (if it is in the map). If @(f x)@ is 'Nothing', the element is deleted.
-- If it is @('Just' y)@, the key @k@ is bound to the new value @y@.
```

Reply:
We can just commit these suggestions for them. They seem relatively inactive.